### PR TITLE
pop: add page

### PR DESCRIPTION
**Pull request by CovertDuck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): v0.2.0**

### CODE REVIEW

File: pages/common/pop.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# pop
+
+> Send emails from your terminal.
+> More information: <https://github.com/charmbracelet/pop>.
+
+- Launch the Text-based User Interface:
+
+`pop`
+
+- Send email with file content as body:
```
Comment (mfrw) [inline pages/common/pop.md:10]: Maybe ?
```suggestion
- Send an email with an attached file:
```

File: pages/common/pop.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`pop < {{path/to/file}} --from "me@example.com" --to "you@example.com" --subject "On the Subject of Ducks..." --attach invoice.pdf`
```
Comment (acuteenvy) [inline pages/common/pop.md:12]:
```suggestion
`pop < {{path/to/message.md}} --from {{me@example.com}} --to {{you@example.com}} --subject "{{On the Subject of Ducks...}}" --attach {{path/to/attachment}}`
```

File: pages/common/pop.md (hunk 1 above)
Comment (CovertDuck) [reply pages/common/pop.md:10]: Something like "Populate Body/Content with specified file"?

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @CovertDuck.

File: pages/common/pop.md (hunk 1 above)
Comment (CovertDuck) [reply pages/common/pop.md:10]: `Use file content to populate the email's body`

@mfrw: Perhaps that would be better?

Also, I'm not sure if the different sections of the email deserve capitalization:
**body → Body
attachment → Attachment
to → To
...**

I will let it like so for the time being.

Comment (acuteenvy) [reply pages/common/pop.md:10]: How about this?
```suggestion
- Send an email using the content of a Markdown file as body:
```

Comment (CovertDuck) [reply pages/common/pop.md:10]: Looks fine to me. Thanks. :)

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**